### PR TITLE
Document Line2D

### PR DESCRIPTION
**Pull request by hpvb**

### CODE REVIEW

File: doc/base/classes.xml [hunk 1]
```diff
@@ -26310,8 +26310,10 @@
 </class>
 <class name="Line2D" inherits="Node2D" category="Core">
 	<brief_description>
+		A 2 dimensional line.
```
Comment (Noshyaar) [inline doc/base/classes.xml]: A *two* would be better? I'm not native speaker but I think that for smaller numbers it should be text?

Comment (willnationsdev) [reply doc/base/classes.xml]: Actually, in this case, since it's math related and referring to 2D, it should be "2-dimensional"

Comment (hpvb) [reply doc/base/classes.xml]: After talking to KidsCanCode we decided to just use '2D'

Comment (turtletooth) [reply doc/base/classes.xml]: I think 2D is more clear.  Grammatically, any value under 10 is generally spelled out, but in this case 2D works better.

Comment (NathanLovato) [reply doc/base/classes.xml]: It's fine to use 2, 3, 4, 5 in general for smaller number - the only exception tends to be "one". It's common to write numbers this way when you're blogging. The reason: it's reads faster for most.

**Pull request merged**